.gitignore

```diff
@@ -1,42 +1,178 @@
-# Python compiled files
-*.py[ocd]
+# Standard gitignore list
 
-# C extensions
-*.so
+# Archive files
+*.7z
+*.bz2
+*.gz
+*.lzma
+*.lzma2
+*.tar
+*.xz
+*.zip
 
-# Kate
-.directory
+# Temporary / backup files
+*.bak
+*.swp
+*.tmp
+*.temp
 
-# PyPi configuration
-.pypirc
+# OS-specific files
+.DS_Store
+Thumbs.db
 
-# Packages
+# Byte-compiled / optimized / DLL files
+__pycache__/
+*.py[cod]
+*$py.class
+
+# C extensions
+*.so
+
+# Distribution / packaging
 *.egg
-*.egg-info
-dist
-build
-eggs
-parts
-bin
-var
-sdist
-develop-eggs
+*.egg-info/
+.eggs/
 .installed.cfg
-lib
-lib64
+.Python
+bin/
+build/
+develop-eggs/
+dist/
+downloads/
+eggs/
+lib/
+lib64/
+MANIFEST
+parts/
+pip-wheel-metadata/
+sdist/
+share/python-wheels/
+var/
+wheels/
+
+# PyInstaller
+#  Usually these files are written by a python script from a template
+#  before PyInstaller builds the exe, so as to inject date/other infos into it.
+*.manifest
+*.spec
+
+# Installer logs
+pip-log.txt
+pip-delete-this-directory.txt
+
+# Unit test / coverage reports
+.cache
+*.cover
+.coverage
+.coverage.*
+.hypothesis/
+.nox/
+.pytest_cache/
+.tox/
+coverage.xml
+htmlcov/
+nosetests.xml
 
 # Translations
 *.mo
 
-# Gedit files
+# Django stuff:
+*.log
+local_settings.py
+db.sqlite3
+db.sqlite3-journal
+
+# Flask stuff:
+instance/
+.webassets-cache
+
+# Scrapy stuff:
+.scrapy
+
+# Sphinx documentation
+doc/_build/
+docs/_build/
+
+# PyBuilder
+target/
+
+# Jupyter Notebook
+.ipynb_checkpoints
+
+# IPython
+profile_default/
+ipython_config.py
+
+# pyenv
+.python-version
+
+# pipenv
+#   According to pypa/pipenv#598, it is recommended to include Pipfile.lock in version control.
+#   However, in case of collaboration, if having platform-specific dependencies or dependencies
+#   having no cross-platform support, pipenv may install dependencies that don't work, or not
+#   install all needed dependencies.
+#Pipfile.lock
```

Comment:
Its there so users can uncomment in situations where the `Pipfile.lock` shouldn't be checked in to version control. However, since it is recommended that it is, and no Spyder-related project, nor any project I'm involved with, uses `pipenv` nor is likely to ever (given its serious problems and decline in maintenance and popularity in favor of better and more modern alternatives like Flit and Poetry,